ios/CodePush/CodePush.m

```diff
@@ -247,6 +247,20 @@ - (instancetype)init
  */
 - (void)initializeUpdateAfterRestart
 {
+    dispatch_async(dispatch_get_main_queue(), ^{
+        if ([_bridge.bundleURL.scheme hasPrefix:@"http"]) {
```

Comment:
I don't think it does ignoreCase comparison, but because we have been using this same check elsewhere, e.g. https://github.com/Microsoft/react-native-code-push/blob/clear-updates-init/ios/CodePush/CodePush.m#L346 and it has not ever caused any problems in practice. The provided URL is generated by RN's app template and there is rarely any need for users to modify it or capitalize the scheme. Hence, I would prefer to leave it as-is.